test/unit/platforms/test_locplatform.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015-2025 Earth Sciences Department, BSC-CNS
+#
+# This file is part of Autosubmit.
+#
+# Autosubmit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Autosubmit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for the Local Platform."""
+
+from autosubmit.platforms.locplatform import LocalPlatform
+
+_EXPID = 't001'
+
+
+def test_local_platform_copy():
+    local_platform = LocalPlatform(_EXPID, 'local', {}, auth_password=None)
+
+    copied = local_platform.create_a_new_copy()
+
+    assert local_platform.name == copied.name
+    assert local_platform.expid == copied.expid
+    assert local_platform.get_checkhost_cmd() == copied.get_checkhost_cmd()
```

Comment:
Mainly to cover `get_checkhost_cmd` :grimacing: , probably increasing coverage in 0.0001%, but poc a poc :grimacing: